hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/CommandDispatcher.java

```diff
@@ -102,7 +98,7 @@ public void handle(SCMCommand command) {
         LOG.error("Exception while handle command, ", ex);
       }
     } else {
-      LOG.error("Unknown SCM Command queued. There is no handler for this " +
+      LOG.warn("Unknown SCM Command queued. There is no handler for this " +
```

Comment:
Why is this not an `ERROR`

Reply:
Thanks @kerneltime  for the comment and review.
No exception is generated here, so I think setting the log level to `warn` is more appropriate here. What do you think?

Reply:
SCM getting an unknown command implies that either something during upgrade failed or a service is misconfigured. I would keep it as `ERROR`

Reply:
+1 for keeping it as error.

Reply:
Thanks @kerneltime  and @adoroszlai  for the explanation.
I have updated it.